modules/bmo-patching-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -0,0 +1,58 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-patching-the-hostfirmwarecomponents-resource_{context}"]
+= Patching the HostFirmwareComponents resource
+
+You can patch the `HostFirmwareComponents` resource for a provisioned host, cordon and drain the host, annotate the host to reboot, and then uncordon the host to make the changes take effect.
```

Comment:
Similar to my previous comment, I am uncertain if I should perform all these actions for the changes to take effect or if any one action will do it.